config/proto/config.proto

```diff
@@ -252,6 +252,38 @@ message ClientConfig {
     // info up to date about each client. This should not be sent too
     // frequently. The default is 1 day (86400 seconds).
     int64 client_info_update_time = 40;
+
+    // Clients may connect to servers which use a self-signed certificate.
+    // This list allows to specify a set of certificate fingerprints (SHA256)
+    // which are used to validate TLS server certificates. If there's no match,
+    // certificate validation will fallback to checking the certificate against the
+    // client's CA file and, eventually, use public CA lists to verify the cert.
+    // Important: to enforce certificate pinning, set `ForceCertificatePinning' to true.
```

Comment:
Maybe also add the following comment:

To obtain the thumbprint of any server for pinning you can use openssl
openssl s_client -connect www.google.com:443  | openssl x509 -fingerprint -sha1

sha1 Fingerprint=00:94:06:0F:B6:4D:E7:50:72:1E:56:17:9B:C5:85:CF:61:78:42:F1

Also maybe support checking the fingerprint in the format openssl provides to make it more ergonomic for users to just copy/paste the output from openssl into the config file and it should work